4-DeviceCodeFlow/Readme.md

```diff
@@ -0,0 +1,236 @@
+---
+services: active-directory
+platforms: dotnet
+author: v-shln
+level: 100
+client: .NET Desktop (Console)
+service: Microsoft Graph
+endpoint: Microsoft identity platform
+page_type: sample
+languages:
+  - csharp  
+products:
+  - azure
+  - azure-active-directory  
+  - dotnet
+  - office-ms-graph
+description: "This sample demonstrates a .NET Desktop (Console) application authenticating with device code flow"
+---
+# Sign-in a user with the Microsoft identity platform using the device code flow and call Microsoft Graph.
+
+![Build badge](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/<BuildNumber>/badge)
+
```

Comment:
request a build number from jean-marc

Reply:
You can add the following build badge:

```md
![.NET Core](https://github.com/Azure-Samples/ms-identity-dotnet-desktop-tutorial/workflows/.NET%20Core/badge.svg)
```

![.NET Core](https://github.com/Azure-Samples/ms-identity-dotnet-desktop-tutorial/workflows/.NET%20Core/badge.svg)

Reply:
Updated the build badge.
Thanks.